### DISCUSSION TITLE
Accuracy showing 99.99% when SS'ing

### DISCUSSION THREAD

**Discussion by shinobu-uwu** [ppy/osu#16228] [Q&A] [answered]
As the title says i was SS'ing the map and the acc in-game showed 99.99%, but I can't reproduce it anymore (the replay always shows 100% and on another tries it was 100% too). Since I have nothing to provide for an issue I wanted to ask if you're aware of this bug or if there is somewhere I can start looking into to provide more information.

**Comment by bdach** [comment]
i can't respond to this in any meaningful way if there is no information provided (no logs, no screenshots, no replay)

**Comment by shinobu-uwu** [comment]
I actually could reproduce, the replay does show it suddenly going from 100% to 99.99% around 40% into the map.
The replay: https://drive.google.com/file/d/1HzDWvfJfGta2qAfwlVaVyoC0a1F5V2jQ/view?usp=sharing

**Reply by shinobu-uwu** [reply]
FWIW I'm playing on arch linux, it was played on wayland but I just saw the replay on X11 and the issue is the same

**Reply by Flamiii** [reply]
I imported the replay into my game and it shows up as a 1x100 99.92% FC as opposed to an SS and you apparently got the 100 ~90% through the map at least for me. I will say that your accuracy can sometimes go from 100% to 99.99% or decrease by very small increments if you miss a slider end and it won't show a 100 so that may be it but I'm not sure.

**Reply by shinobu-uwu** [reply]
Yeah that's the reason, in the score screen it shows one slider end missed and the only 100 I got was from a circle

**Comment by shinobu-uwu** [comment, marked as answer]
As @Flamiii pointed out the reason was I missed a slider end in the middle of the map, though I think it should somehow be showed better to the user this closes the discussion.
